doc/reference/reference_lua/net_box.rst

```diff
@@ -533,7 +535,60 @@ Below is a list of all ``net.box`` functions.
             - B
             ...
 
+    ..  _conn-watch:
 
+    ..  method:: watch(key, func)
+
+        Subscribe to events broadcast by a remote host.
+
+        :param string key: a key name of an event to subscribe to
+        :param function func:  a callback to invoke when the key value is updated
+        :return: a watcher handle that can be used to unregister the watcher
+
+        Using the ``watch`` method of a net.box connection, you can subscribe to events broadcast by a remote host.
+        The method has the same syntax as the :doc:`box.watch() </reference/reference_lua/box_watchers/broadcast>`
+        function, which is used for subscribing to events locally.
+
+        A watcher callback is first invoked unconditionally after the watcher registration.
+        Subsequent invocations are triggered by :doc:`box.broadcast() </reference/reference_lua/box_watchers/broadcast>`
+        called on the remote host.
+        A watcher callback is passed the name of the key for which it was registered and the current key data.
+        A watcher callback is always invoked in a new fiber so it is okay to yield in it.
+        A watcher callback is never executed in parallel with itself.
+        If the key is updated while the watcher callback is running, the callback will be invoked again with the new
+        value as soon as it returns.
```

Comment:
Maybe, instead of duplicating the watcher callback properties in three places, better describe them in one place and add a reference to all other places?